Multisig.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -33,44 +33,53 @@
         "repositoryURL": "https://github.com/firebase/boringssl-SwiftPM.git",
         "state": {
           "branch": null,
-          "revision": "7bcafa2660bc58715c39637494550d1ed7cd7229",
-          "version": "0.0.7"
+          "revision": "15889fadef7078dfa1392b0e2b08f3d0e9797104",
+          "version": "0.7.1"
         }
       },
       {
         "package": "CryptoSwift",
         "repositoryURL": "https://github.com/krzyzanowskim/CryptoSwift.git",
         "state": {
           "branch": null,
-          "revision": "79ac632c85df81f2618dd838fc3869633e163370",
-          "version": "1.2.0"
+          "revision": "3a2acbb32ab68215ee1596ee6004da8e90c3721b",
+          "version": "1.0.0"
```

Comment:
why was it downgraded? Could you please update in your package `.package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", from: "1.0.0")` to 1.4.0?

Reply:
probably it will be properly resolved without the update too

Reply:
At the moment we can't update the CryptoSwift package.

```Because no versions of EthereumAddress match the requirement 1.3.1..<2.0.0 and EthereumAddress 1.3.0 depends 
on CryptoSwift 1.0.0..<1.1.0, EthereumAddress >=1.3.0 requires CryptoSwift 1.0.0..<1.1.0.
And because root depends on CryptoSwift 1.4.0..<2.0.0 and root depends on EthereumAddress 1.3.0..<2.0.0, 
version solving failed.
```

As I understood this doesn't cause an issue for the gnosis project.  We will work on the update soon